server/src/messages/dto/create-message.dto.ts

```diff
@@ -1,9 +1,15 @@
-import { PartialType } from '@nestjs/mapped-types';
-import { User } from 'src/users/entities/user.entity';
-import { Message } from '../entities/message.entity';
+import { IsNotEmpty, IsEnum, IsOptional } from 'class-validator';
 
-export class CreateMessageDto extends PartialType(Message) {
+import { User } from '../../users/entities/user.entity';
+import { Transaction } from '../../transactions/entities/transaction.entity';
+
+export class CreateMessageDto {
+  @IsNotEmpty()
   text: string;
+
+  @IsNotEmpty()
   user: User;
```

Comment:
### recommendation
Same as previous comment. It can be derived from the signed in user

Reply:
nice! i like this move
i am going to spin it off into another ticket though if that's ok
this is strictly a BE PR and idk how to send a cookie with a curl request
https://github.com/Nonprofit-Exchange-Hub/web-app/issues/152